### ISSUE TITLE
All folders deleted by a user accidentaly

### ISSUE THREAD

**Issue by cmarbach** [nilsteampassnet/TeamPass#2180]
### Steps to reproduce
1. Define a basic folder structure; folder1 and folder2, both under the root level
2. Define 2 roles, role1 and role2, first has write permissions on folder1 and second has write permissions on folder2, no access otherwise
3. Create a user and assign him role1
4. Login as that user and create an item under folder1, save, everything is fine
5. Select you personal folder
6. menu "Delete a folder" - personal folder is selected even if it's grayed out
7. confirm deletion, all folders are gone !
8. as admin, verify that even folder2 has been deleted, you may find all deleted folders and items in the menu Utilities/Deletion

PS: feature "personal folder" is active

### Expected behaviour
Deletion of personal folder should be forbidden and at least deletion of all folders should not happen.

### Actual behaviour
All folders and items are deleted (including folders to which access is not granted and items under the personal folder).

### Server configuration
**Operating system**:
CentOS 7.4.1708 - 3.10.0-693.21.1.el7.x86_64
**Web server:**
Apache2
**Database:**
MySQL
**PHP version:**
php56w - 5.6.33-1.w7.x86_64
**Teampass version:** 
2.1.26
**Teampass configuration file:**

**Updated from an older Teampass or fresh install:**
PLEASE attach to this message the file `/includes/config/tp.config.php`.

### Client configuration
**Browser:**
Chrome 63, Internet Explorer 11
**Operating system:**
Windows 8.1
#### Web server error log
```
Insert your webserver log here
```

#### Log from the web-browser developer console (CTRL + SHIFT + i)
```
Insert the log here and especially the answer of the query that failed.
```

**Comment by cmarbach** [comment]
<?php
global $SETTINGS;
$SETTINGS = array (
    'max_latest_items' => '10',
    'enable_favourites' => '1',
    'show_last_items' => '1',
    'enable_pf_feature' => '1',
    'log_connections' => '1',
    'log_accessed' => '1',
    'time_format' => 'H:i:s',
    'date_format' => 'd/m/Y',
    'duplicate_folder' => '0',
    'item_duplicate_in_same_folder' => '0',
    'duplicate_item' => '1',
    'number_of_used_pw' => '0',
    'manager_edit' => '1',
    'cpassman_dir' => '/var/www/teampass',
    'cpassman_url' => 'https://teampass.X.X',
    'favicon' => 'https://teampass.X.X/favicon.ico',
    'path_to_upload_folder' => '/var/www/teampass/upload',
    'url_to_upload_folder' => 'https://teampass.X.X/upload',
    'path_to_files_folder' => '/var/www/teampass/files',
    'url_to_files_folder' => 'https://teampass.X.X/files',
    'activate_expiration' => '0',
    'pw_life_duration' => '0',
    'maintenance_mode' => '0',
    'enable_sts' => '0',
    'encryptClientServer' => '1',
    'cpassman_version' => '2.1.26',
    'ldap_mode' => '1',
    'ldap_type' => 'windows',
    'ldap_suffix' => '@X.X.X',
    'ldap_domain_dn' => 'DC=X,DC=X,DC=X',
    'ldap_domain_controler' => 'X.X.X.X,X.X.X.X',
    'ldap_user_attribute' => '0',
    'ldap_ssl' => '0',
    'ldap_tls' => '0',
    'ldap_elusers' => '1',
    'richtext' => '0',
    'allow_print' => '0',
    'roles_allowed_to_print' => '19;23',
    'show_description' => '1',
    'anyone_can_modify' => '0',
    'anyone_can_modify_bydefault' => '0',
    'nb_bad_authentication' => '0',
    'utf8_enabled' => '1',
    'restricted_to' => '0',
    'restricted_to_roles' => '0',
    'enable_send_email_on_user_login' => '0',
    'enable_user_can_create_folders' => '1',
    'insert_manual_entry_item_history' => '0',
    'enable_kb' => '0',
    'enable_email_notification_on_item_shown' => '0',
    'enable_email_notification_on_user_pw_change' => '0',
    'custom_logo' => '',
    'custom_login_text' => '',
    'default_language' => 'french',
    'send_stats' => '0',
    'get_tp_info' => '1',
    'send_mail_on_user_login' => '0',
    'nb_items_by_query' => '10',
    'enable_delete_after_consultation' => '0',
    'enable_personal_saltkey_cookie' => '0',
    'personal_saltkey_cookie_duration' => '31',
    'email_smtp_server' => 'X.X.X',
    'email_smtp_auth' => '0',
    'email_auth_username' => '',
    'email_auth_pwd' => '',
    'email_port' => '25',
    'email_server_url' => 'https://teampass.X.X',
    'email_from' => 'info@X.X',
    'email_from_name' => 'TEAMPASS',
    'pwd_maximum_length' => '128',
    'delay_item_edition' => '15',
    'allow_import' => '0',
    'proxy_ip' => '',
    'proxy_port' => '',
    'upload_maxfilesize' => '10mb',
    'upload_docext' => 'doc,docx,dotx,xls,xlsx,xltx,rtf,csv,txt,pdf,ppt,pptx,pot,dotx,xltx',
    'upload_imagesext' => 'jpg,jpeg,gif,png',
    'upload_pkgext' => '7z,rar,tar,zip',
    'upload_otherext' => 'sql,xml',
    'upload_imageresize_options' => '1',
    'upload_imageresize_width' => '800',
    'upload_imageresize_height' => '600',
    'upload_imageresize_quality' => '90',
    'use_md5_password_as_salt' => '0',
    'ga_website_name' => '',
    'api' => '0',
    'subfolder_rights_as_parent' => '1',
    'show_only_accessible_folders' => '1',
    'enable_suggestion' => '0',
    'otv_expiration_period' => '7',
    'send_stats_time' => '0',
    'tree_counters' => '0',
    'item_extra_fields' => '0',
    'enable_attachment_encryption' => '0',
    'copy_to_clipboard_small_icons' => '1',
    'timezone' => 'Europe/Zurich',
    'settings_offline_mode' => '0',
    'offline_key_level' => '0',
    'bck_script_filename' => 'bck_cpassman',
    'bck_script_path' => '/var/www/teampass/backups',
    'can_create_root_folder' => '0',
    'ldap_bind_dn' => 'X.X.X.X',
    'menu_type' => 'context',
    'email_security' => 'none',
    'default_session_expiration_time' => '60',
    'duo' => '0',
    'enable_server_password_change' => '0',
    'syslog_enable' => '0',
    'syslog_host' => 'localhost',
    'syslog_port' => '514',
    'google_authentication' => '0',
    'ldap_object_class' => '0',
);

**Comment by cmarbach** [comment]
[tp.config.txt](https://github.com/nilsteampassnet/TeamPass/files/1859263/tp.config.txt)

**Comment by nilsteampassnet** [comment]
It is a known issue in 2.1.26
It was fixed in 2.1.27 with #1608
